packages/runtime/src/widgets/Form/__tests__/Checkbox.test.tsx

```diff
@@ -0,0 +1,174 @@
+              value: false,
+            },
+          },
+          {
+            name: "Button",
+            properties: {
+              label: "Set Value",
+              onTap: {
+                executeCode: "checkbox.setValue(true)",
+              },
+            },
+          },
+          ...defaultFormButton,
+        ]}
+        id="form"
+      />,
+      { wrapper: FormTestWrapper },
+    );
+
+    const setValueButton = screen.getByText("Set Value");
+    const getValueButton = screen.getByText("Get Value");
+    fireEvent.click(setValueButton);
+    fireEvent.click(getValueButton);
+
+    await waitFor(() => {
+      expect(logSpy).toHaveBeenCalledWith(
+        expect.objectContaining({ checkbox: true }),
+      );
+    });
+  });
+
+  test("updates when binding changes value", async () => {
+    const logSpy = jest.spyOn(console, "log");
+    render(
+      <Form
+        children={[
+          {
+            name: "Checkbox",
+            properties: {
+              id: "binding",
+              label: "Check Box",
+              value: `\${ensemble.storage.get('binding')}`,
+            },
+          },
+          {
+            name: "Button",
+            properties: {
+              label: "Set Value",
+              onTap: {
+                executeCode: "ensemble.storage.set('binding',true)",
+              },
+            },
+          },
+          ...defaultFormButton,
+        ]}
+        id="form"
+      />,
+      { wrapper: FormTestWrapper },
+    );
+
+    const setValueButton = screen.getByText("Set Value");
+    const getValueButton = screen.getByText("Get Value");
+    fireEvent.click(setValueButton);
+
+    await waitFor(() => {
+      fireEvent.click(getValueButton);
+      expect(logSpy).toHaveBeenCalledWith(
+        expect.objectContaining({ binding: true }),
+      );
+    });
+  });
+
+  test("binding change overwrites user input value", async () => {
+    const logSpy = jest.spyOn(console, "log");
+    render(
+      <Form
+        children={[
+          {
+            name: "Checkbox",
+            properties: {
+              id: "userInput",
+              label: "Check Box",
+              value: `\${ensemble.storage.get('userInput') ?? true}`,
+            },
+          },
+          {
+            name: "Button",
+            properties: {
+              label: "Set Value",
+              onTap: {
+                executeCode: "ensemble.storage.set('userInput',false)",
+              },
+            },
+          },
+          ...defaultFormButton,
+        ]}
+        id="form"
+      />,
+      { wrapper: FormTestWrapper },
+    );
+
+    const setValueButton = screen.getByText("Set Value");
+    const getValueButton = screen.getByText("Get Value");
+    fireEvent.click(setValueButton);
+
+    await waitFor(() => {
+      fireEvent.click(getValueButton);
+      expect(logSpy).toHaveBeenCalledWith(
+        expect.objectContaining({ userInput: false }),
+      );
```

Comment:
You should also assert that the html element has the true attribute. The primary purpose of these tests is to verify what the user sees.